packages/vscode-extension/test/extension/utils/prerelease.test.ts

```diff
@@ -0,0 +1,100 @@
+import * as sinon from "sinon";
+import * as chai from "chai";
+import { PrereleasePage } from "../../../src/utils/prerelease";
+import { ExtensionContext, Memento } from "vscode";
+import mockedEnv, { RestoreFn } from "mocked-env";
+import { ExtTelemetry } from "../../../src/telemetry/extTelemetry";
+import * as spies from "chai-spies";
+chai.use(spies);
+const spy = chai.spy;
+const ShowWhatIsNewNotification = "show-what-is-new-notification";
+const reporterSpy = spy.interface({
+  sendTelemetryEvent(
+    eventName: string,
+    properties?: { [p: string]: string },
+    measurements?: { [p: string]: number }
+  ): void {},
+});
+function gloablStateKeys(): readonly string[] {
+  return ["PrereleaseState.Version"];
+}
+
+function globalStateUpdate(key: string, value: any): any {}
+
+function globalStateGet(key: string): string {
+  return "0.0.0";
+}
+
+describe("versionUtil", () => {
+  const sandbox = sinon.createSandbox();
+  let mockedEnvRestore: RestoreFn;
+  let context: ExtensionContext;
+  const mockGlobalState: Memento = {
+    keys: gloablStateKeys,
+    get: globalStateGet,
+    update: globalStateUpdate,
+  };
+  before(() => {
+    chai.util.addProperty(ExtTelemetry, "reporter", () => reporterSpy);
+  });
+  beforeEach(() => {
+    mockedEnvRestore = mockedEnv({ TEAMSFX_V3: "true" });
```

Comment:
You can use the code below just like other test cases instead of using another library

```
sandbox.stub(commonTools, "isV3Enabled").returns(true);
```

Reply:
Not blocking issue, you can merge and fix in another pr

Reply:
thanks, will update the code this PR.